docs/semgrep-app/notifications.md

```diff
@@ -59,8 +59,168 @@ To view, add, remove, disable, or enable your saved channels:
+   <td>Send messages as @Semgrep.
+   </td>
+   <td>Enables the Semgrep Slack app to send findings to channels.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/chat:write.customize">chat:write.customize</a></strong>
+   </td>
+   <td>Send messages as @Semgrep with a customized username and avatar.
+   </td>
+   <td>Helps users identify Semgrep Slack app messages through the use of an image and username.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/chat:write.public">chat:write.public</a></strong>
+   </td>
+   <td>Send messages to channels <code>@Semgrep</code> isn't a member of.
+   </td>
+   <td>Enables users to invoke Semgrep Slack app features in any public channel using the slash command.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/commands">commands</a></strong>
+   </td>
+   <td>Add shortcuts or slash commands that people can use.
+   </td>
+   <td>Enables the Semgrep Slack app to register custom slash commands such as <code>/semgrep_subscribe</code> used for notification subscription.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/emoji:read">emoji:read</a></strong>
+   </td>
+   <td>View custom emoji in a workspace.
+   </td>
+   <td>Allows Semgrep to support a workspace's custom emojis.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/im:write">im:write</a></strong>
+   </td>
+   <td>Start direct messages with people.
+   </td>
+   <td>Allows users to interact with the Semgrep Slack app and use the slash commands in direct messages.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/links:write">links:write</a></strong>
+   </td>
+   <td>Show previews of URLs in messages.
+   </td>
+   <td>Enables Semgrep Slack app to include links in messages.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/users:read">users:read</a></strong>
+   </td>
+   <td>View profile details about people in a workspace.
+   </td>
+   <td>Enables Semgrep Slack app to correctly address users in messages.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/users:write">users:write</a></strong>
+   </td>
+   <td>Set presence for Semgrep
+   </td>
+   <td>Used by the Semgrep Slack app to interact with the workspace and allows the users to add it in the relevant channels.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/workflow.steps:execute">workflow.steps:execute</a></strong>
+   </td>
+   <td>Add steps that people can use in Workflow Builder.
+   </td>
+   <td>Enables Semgrep to make use of modals and drop-down boxes when a user sets up notifications.
+   </td>
+  </tr>
+  <tr>
+   <td><strong><a href="https://api.slack.com/scopes/groups:read">groups:read</a></strong>
+   </td>
+   <td>View basic information about private channels that your Slack app has been added to.
```

Comment:
@vivekkhimani I thought Semgrep won't be used in private channels at all? Just checking.

Reply:
@s-santillan actually it's being used. We recently had a customer who wanted to set it up in the private channels and it worked as expected.

Reply:
Okay!